portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -82,6 +86,14 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       String addedBy = (String) entity.getProperty("addedBy");
       Date addedDate = (Date) entity.getProperty("addedDate");
 
+      // If a languageCode was specified, translate the comments' message in that language
+      if(languageCode != null) {
+        Translate translate = TranslateOptions.getDefaultInstance().getService();
+        Translation translation =
+            translate.translate(message, Translate.TranslateOption.targetLanguage(languageCode));
```

Comment:
What if the language code provided in the url is not supported? What happens in this function call in this case? This is similar to safeguards you put earlier for the max-comments parameter.

Reply:
You are right, I added a safeguard to make sure the language code is in the ISO languages list (they are all supported by the API). Thank you a lot!